solidity/dashboard/src/sagas/coverage-pool.js

```diff
@@ -399,6 +406,46 @@ export function* subscribeToWithdrawalCompletedEvent() {
   }
 }
 
+export function* subscribeToAuctionCreatedEvent() {
+  const requestChan = yield actionChannel(
+    RISK_MANAGER_AUCTION_CREATED_EVENT_EMITTED
+  )
+
+  while (true) {
+    yield take(requestChan)
+
+    const hasRiskManagerOpenAuctions = yield call(
+      Keep.coveragePoolV1.hasRiskManagerOpenAuctions
+    )
+
+    yield put(
+      covTokenUpdated({
+        hasRiskManagerOpenAuctions,
+      })
+    )
+  }
+}
+
+export function* subscribeToAuctionClosedEvent() {
+  const requestChan = yield actionChannel(
+    RISK_MANAGER_AUCTION_CLOSED_EVENT_EMITTED
+  )
+
+  while (true) {
+    yield take(requestChan)
+
+    const hasRiskManagerOpenAuctions = yield call(
+      Keep.coveragePoolV1.hasRiskManagerOpenAuctions
+    )
+
+    yield put(
+      covTokenUpdated({
+        hasRiskManagerOpenAuctions,
+      })
+    )
+  }
+}
+
```

Comment:
Let's add unit tests for these sagas.

Reply:
27e566d66d3f2927b7ed6dc7672c137ba2db746e